dts-bundle-generator.config.ts

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I just added this to the eslint ignore since we're ignoring other config files and this had quite a few violations.